src/components/Banner/Banner.tsx

```diff
@@ -95,35 +136,19 @@ const Banner: React.FC<BannerProps> = ({
       >
         <ContentWrapper paddingSize={SpaceSizes.md}>
           <Inline align="flex-start" gap={SpaceSizes.xl} stretch={1}>
-            <Stack align="center" gap={SpaceSizes.md}>
-              <Text $variant={variant} as="div" size={TextSizes.md}>
-                {children}
-              </Text>
-              {isNonEmptyArray(actions) && (
-                <Inline gap={SpaceSizes.mdPlus}>
-                  {actions.map((action) => (
-                    <StyledButton
-                      key={action.name}
-                      $variant={variant}
-                      color={ButtonColors.secondary}
-                      href={
-                        (action as AbsoluteLinkActionKind<[React.MouseEvent]>)
-                          .href
-                      }
-                      name={action.name}
-                      to={
-                        (action as RelativeLinkActionKind<[React.MouseEvent]>)
-                          .to
-                      }
-                      variant={ButtonVariants.text}
-                      onClick={action.onClick}
-                    >
-                      {action.label}
-                    </StyledButton>
-                  ))}
-                </Inline>
-              )}
-            </Stack>
+            {isInline ? (
+              <Inline align="flex-start" gap={SpaceSizes.md}>
```

Comment:
please add `justify: space-between` right now actions sticks to the content and it looks strange